pom.xml

```diff
@@ -56,8 +56,10 @@
             </dependency>
             <dependency>
                 <groupId>com.vaadin</groupId>
-                <artifactId>vaadin-spring</artifactId>
+                <artifactId>vaadin-bom</artifactId>
```

Comment:
let us dont use the platform dependency here, as we are going to include this project to platform, this will cause circular dependency issue

Reply:
Would using `vaadin-spring-bom` instead avoid the circular dependency?

Reply:
unfortunately, no. Both `vaadin-bom` and `vaadin-spring-bom` are generated by platform. 
so in this case, we better just use direct dependencies. for example 
```
        <dependency>
            <groupId>com.vaadin</groupId>
            <artifactId>license-checker</artifactId>
            <version>1.10.0</version>
        </dependency>
```

Reply:
Done, thanks!